### ISSUE TITLE
All OSX runs failing in CI

### ISSUE THREAD

**Issue by stephentoub** [dotnet/corefx#8166]
```
    11 Warning(s)
    0 Error(s)

Time Elapsed 00:33:32.84
Build Exit Code = 0
[osx_debug_prtest] $ /bin/sh -xe /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/hudson5854860390763282096.sh
+ tar -czf bin/build.tar.gz bin/AnyOS.AnyCPU.Debug bin/Linux.AnyCPU.Debug bin/OSX.AnyCPU.Debug bin/OSX.x64.Debug bin/Unix.AnyCPU.Debug bin/Windows_NT.AnyCPU.Debug bin/ref bin/packages '--exclude=*.Tests'
tar: --exclude=*.Tests: Cannot stat: No such file or directory
tar: Error exit delayed from previous errors.
Build step 'Execute shell' marked build as failure
```

**Comment by mellinoe** [comment]
Is this just because the "--exclude" parameter is quoted?

**Comment by ellismg** [comment]
I thought that @mmitche tried to fix this with #8160 not fix this?  Matt, do you know why the jobs did not seem to get updated?

**Comment by Priya91** [comment]
The issue doesn't seem to repro with latest runs now.. the failure must have happened before the generator created new job definition.
